packages/shared/lib/contexts/governance/utils/createProposalsFromEvents.ts

```diff
@@ -15,7 +15,7 @@ export async function createProposals(): Promise<IProposal[]> {
     return proposals
 }
 
-async function createProposalFromEvent(event: Event): Promise<IProposal> {
+export async function createProposalFromEvent(event: Event): Promise<IProposal> {
```

Comment:
Hmm we should probably separate this file into multiple functions to help with clarity: `createProposals` and `createProposalFromEvent`. The current filename doesn't have a corresponding function though anyway